client/src/components/rr_components/OverallRatings.jsx

```diff
@@ -1,7 +1,58 @@
 import React from 'react';
+import _ from 'underscore';
 
+<<<<<<< HEAD
+const OverallRatings = ({ data }) => {
+  const count = parseInt(data.recommended.false, 10) + parseInt(data.recommended.true, 10);
+  let allRatings = 0;
+  _.each(data.ratings, (rating, i) => {
+    allRatings += rating * i;
+  });
+  const averageRating = Math.round((allRatings / count) * 10) / 10;
```

Comment:
@avaboonzaayer Could you add the average rating at the App level so we could also all get access to it?

Reply:
Doesn't have to be now, just next time when you submit a pull request